src/launch/launch.scss

```diff
@@ -1,3 +1,110 @@
 @import "styles/colors.scss";
+@import "styles/styles.scss";
 .page.launch {
+  display: grid;
+  justify-content: center;
+  grid-gap: 30px;
+  .banner {
+    .banner1 {
+      // Make like a heading
+      @extend .heading;
+      font-size: 4vw;
```

Comment:
hmmm, haven't used vw for a font size before.  but it could make for easy responsiveness.  gotta see that

Reply:
yes, responsive font sizes (thinking about mobile)